gatherling/models/Event.php

```diff
@@ -1826,9 +1756,9 @@ public function updateDecksFormat($format)
 
     public function startEvent()
     {
-        $entries = $this->getRegisteredEntries();
+        $entries = $this->getValidRegisteredEntries();
         Standings::startEvent($entries, $this->name);
-        $this->dropInvalidEntries();
+        // $this->dropInvalidEntries();
```

Comment:
This will leave invalid entries in the event, where they can sneak into an event late by fixing their decklist.
This is not the desired behaviour.

Reply:
I have merge the drop function with getRegisteredEntries function